docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Teleport with GitHub.
+description: How to use SSH key extensions with Teleport and GitHub.
+h1: Use SSH Key extensions.
+---
+
+## Prerequisites
+
+- Teleport v(=teleport.version) Open Source or Enterprise.
+
+## Step 1. Import the Teleport CA into Github
+
+In order to export the Teleport CA, execute
+
+```code
+tctl auth export > teleport.ca
+```
+
+Then follow the steps outlined[here](https://docs.github.com/en/organizations/managing-git-access-to-your-organizations-repositories/managing-your-organizations-ssh-certificate-authorities),
+to import the exported Teleport CA
+
+## Step 2. Configure key extensions to be included in key generation.
```

Comment:
```suggestion
## Step 2. Configure key extensions to be included in key generation
```

Reply:
Let's choose a more succinct heading here. Once there's a description at the beginning of this guide of what the reader will learn and why it is useful, it should be easier to make this heading shorter, since it won't be doing as much descriptive work.